src/layout/Breadcrumbs.tsx

```diff
@@ -9,37 +7,47 @@ import { Breadcrumbs as MuiBreadcrumbs } from '@material-ui/core';
 import NavigateNextIcon from '@material-ui/icons/NavigateNext';
 
 import { mapBreadcrumbName } from '../utils/mapBreadcrumbName';
+import styled from 'styled-components';
+
+const StyledBreadcrumbs = styled.div`
+  grid-area: breadcrumbs;
+  padding: 0.5rem;
+
+  a {
+    color: ${({ theme }) => theme.palette.text.secondary};
+  }
+`;
 
 const Breadcrumbs: React.FC = () => {
   const location = useLocation();
   const { t } = useTranslation();
 
   const pathNames = location.pathname.split('/').filter(x => x);
 
+  if (pathNames.length === 0) {
+    return null;
+  }
+
```

Comment:
er dette bedre enn && i return?

Reply:
Skal ikke være noen praktisk forskjell, men personlig syns jeg det er mer lesbart å ta den type ting utenfor hoved-return for å minimere logikk og innrykk i return. 
Uenig?

Reply:
jeg er uenig ja. jeg mener at hvis det er noe som skal rendres, så skal det i return. uavhengig om det er null eller en komponent. Hva mener dere andre, @Eiriknil @placbo ?

Reply:
Usikker. Utstrakt bruk av forkorende notasjoner som &&  & !! ... er fremdeles uvant for meg.

Reply:
tar &&